modules/extensions/src/api/exec.ts

```diff
@@ -0,0 +1,44 @@
+import { extensionPort, proxy } from "../util/comlink";
+import { ExecArgs } from "../types";
+
+export default async function exec({
+  splitStderr,
+  args,
+  env,
+  onOutput = () => {},
+  onStdErr = () => {},
+  onError = () => {},
+  onEnd = () => {},
+}: ExecArgs): Promise<() => void> {
+  let outputStr: string = "";
+
+  // TODO: Make dispose() not cause a BSOD after being called immediately
+  const { dispose, promise } = await extensionPort.experimental.exec(
+    proxy({
+      args: Array.isArray(args) ? args : ["bash", "-c", args],
```

Comment:
is `args` automatically escaped here?

if you enter `git commit -m "test"`, does that break due to the " or not?